### PR TITLE
Temporary fix for delete modal

### PR DESCRIPTION
**Pull request by tahmidefaz**
Delete group modal gets stuck in loading, people want it to work ASAP. I think there are some issues with the groupReducer. I simply disabled it for the quick fix, this is the easiest way I could think of

### CODE REVIEW

**Comment by tahmidefaz** [comment]
The group info load wait time is minute, and I think this whole modal will be replaced once UX completes the new mockups.

File: src/smart-components/group/remove-group-modal.js [hunk 1]
```diff
@@ -87,9 +87,9 @@ RemoveGroupModal.propTypes = {
   closeUrl: PropTypes.string
 };
 
-const mapStateToProps = ({ groupReducer: { selectedGroup, isRecordLoading }}) => ({
+const mapStateToProps = ({ groupReducer: { selectedGroup }}) => ({
   group: selectedGroup,
-  isLoading: isRecordLoading
+  isLoading: false
```
Comment (karelhala) [inline src/smart-components/group/remove-group-modal.js]: Can you add `isLoading: !selectedGroup.loaded` in here?

**Comment by karelhala** [comment]
You'll also have to change how this is tested, to fully go trough the process of removing group:
```diff
diff --git a/src/test/smart-components/group/remove-group-modal.test.js b/src/test/smart-components/group/remove-group-modal.test.js
index dbc0040..eef6895 100644
--- a/src/test/smart-components/group/remove-group-modal.test.js
+++ b/src/test/smart-components/group/remove-group-modal.test.js
@@ -39,6 +39,9 @@ describe('<RemoveGroupModal />', () => {
         group: {
           name: 'Foo',
           uuid: '1'
+        },
+        selectedGroup: {
+          loaded: true
         }
       }
     };

```

**Comment by tahmidefaz** [comment]
@karelhala Great! Looks like the tests are going to pass this time. Just removed an unnecessary change that sneaked with my previous commit.

**Pull request merged**